community/fdm/build

```diff
@@ -2,9 +2,12 @@
 
 patch -p1 < musl.patch
 
+aclocal
+automake
```

Comment:
Can this simply be: `autoreconf` or `autoreconf -i`?

Reply:
`autoreconf` seems to be enough. Done!